3_navigation.ipynb

```diff
@@ -109,26 +226,73 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Return to the RViz screen and give \"2D Nav Goal\" to move the HSR autonomously.\n",
+    "\n",
+    "![title](./imgs/3_move_avoiding_human.png)\n",
+    "\n",
+    "You should see the robot move autonomously while avoiding person."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# Pythonスクリプトからの自律移動"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Autonomous move by Python script"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "前章はrvizから自律移動を実行しましたが、今回はPythonから指定した座標に自律移動する指令を出してみましょう。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
    "source": [
-    "### 1. 準備\n",
+    "In the previous section, autonomous movement was executed from RViz. This time, let's issue a command using the Python script."
```

Comment:
"In the previous section, autonomous movement was executed from RViz. This time, let's issue a command using a Python script."